docs/tutorials/setup-opentelemetry.md

```diff
@@ -24,16 +24,18 @@ yarn --cwd packages/backend add @opentelemetry/sdk-node \
 
 ## Configure
 
-In your `packages/backend/src` folder, create an `instrumentation.ts` file.
```

Comment:
Not sure why it has to be outside /src, but Docker can't find it otherwise. Maybe someone could explain this

Reply:
@SonilPro The .dockerignore file ignores packages/*/src.  An exclusion for the file in the dockerignore could be added using !packages/backend/src/instrumentation.ts

Reply:
Ahh okay. That makes sense. I can move the file to source then if it makes sense, but we could also keep it outside.